bbq/vm/test/vm_test.go

```diff
@@ -3272,3 +3272,120 @@ func TestFunctionPostConditions(t *testing.T) {
 	})
 
 }
+
+func TestDefaultFunctionsWithConditions(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("default in parent, conditions in child", func(t *testing.T) {
+		t.Parallel()
+
+		storage := interpreter.NewInMemoryStorage(nil)
+
+		activation := sema.NewVariableActivation(sema.BaseValueActivation)
+		activation.DeclareValue(stdlib.PanicFunction)
+		activation.DeclareValue(stdlib.NewStandardLibraryStaticFunction(
+			"log",
+			sema.NewSimpleFunctionType(
+				sema.FunctionPurityView,
+				[]sema.Parameter{
+					{
+						Label:          sema.ArgumentLabelNotRequired,
+						Identifier:     "value",
+						TypeAnnotation: sema.AnyStructTypeAnnotation,
+					},
+				},
+				sema.VoidTypeAnnotation,
+			),
+			"",
+			nil,
+		))
+
+		var logs []string
+		vmConfig := &vm.Config{
+			Storage:        storage,
+			AccountHandler: &testAccountHandler{},
+			//ImportHandler: func(location common.Location) *bbq.Program[opcode.Instruction] {
+			//	program, ok := programs[location]
+			//	if !ok {
+			//		assert.FailNow(t, "invalid location")
+			//	}
+			//	return program.Program
+			//},
+			//ContractValueHandler: func(_ *vm.Config, location common.Location) *vm.CompositeValue {
+			//	contractValue, ok := contractValues[location]
+			//	if !ok {
+			//		assert.FailNow(t, "invalid location")
+			//	}
+			//	return contractValue
+			//},
```

Comment:
Is this a TODO? Should it be uncommented or maybe removed?

Reply:
Yeah, it is not needed. Removed